ingestion/src/metadata/ingestion/source/messaging/kafka/connection.py

```diff
@@ -60,6 +69,14 @@ def get_connection(
         if connection.saslMechanism:
             connection.consumerConfig["sasl.mechanism"] = connection.saslMechanism.value
 
+        if (
+            connection.consumerConfig.get("security.protocol") is None
```

Comment:
is not None?

Reply:
`is None` is intended, let's say user has already provided the `security.protocol` in consumer config then that would be given precedence over the new field that has been added.